### ISSUE TITLE
Multilayer Perceptron

### ISSUE THREAD

**Issue by prithagupta** [prithagupta/ML-ILD#1]
- [x] Create the Multilayer Perceptron using Keras/Tensorflow
- [ ] Make this model compatible with [BayesSearchCV](https://scikit-optimize.github.io/stable/modules/generated/skopt.BayesSearchCV.html)
- [ ] Test the model with a synthetic or a RealOpenSSL dataset
- [ ] Include the class_weight parameter, or use weighted loss function same as that used in [MLPClassifier](https://scikit-learn.org/stable/modules/generated/sklearn.neural_network.MLPClassifier.html)
- [ ] Document the Classes
